### DISCUSSION TITLE
How to print the output operation in python.

### DISCUSSION THREAD

**Discussion by Dhrubajyo** [community/community#63616] [Programming Help]
### Body

I have defined two variables to add them. But it is showing an error.

Code  
```
a = 5068978
b = 52
output = print("THe answer is" =  a + b)
```

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by DhrubajyotiBiswasfounderofDribll** [comment]
The code will be:
a = 5068978
b = 52
output = print(f"The answer is" {a} + {b})

You have to use an fstring and assign the variables in curly brackets.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
